src/index.js

```diff
@@ -94,6 +96,10 @@ async function performFetch(network, req, res){
   // parse JSON
   const data = JSON.parse(rawData)
 
+  if (!data.result && !data.error) {
```

Comment:
There are some infura endpoints that can intentionally return a `false` result: https://infura.io/docs/ethereum/json-rpc/eth_mining, https://infura.io/docs/ethereum/json-rpc/eth_submitWork, https://infura.io/docs/ethereum/json-rpc/eth_syncing

Should we do `!(data.result === null || data.result === undefined)` or similar?